plugins/cassandra/cassandraembed.cpp

```diff
@@ -2538,6 +2566,23 @@ class AttributeMapColumnMapper : implements CassandraColumnMapper
     }
 } attributeMapColumnMapper;
 
+class QueryTextColumnMapper : public StringColumnMapper
+{
+public:
+    virtual IPTree *toXML(IPTree *row, const char *name, const CassValue *value)
+    {
+        // Name is "Query/Text ...
+        IPTree *query = row->queryPropTree("Query");
+        if (!query)
+        {
+            query = createPTree("Query");
+            row->setPropTree("Query", query);
```

Comment:
Does this need to say query = row->....? for the situation where it is adding to a remote ptree